src/main/java/de/hpi/bpt/argos/core/ArgosImpl.java

```diff
@@ -0,0 +1,126 @@
+package de.hpi.bpt.argos.core;
+
+import de.hpi.bpt.argos.api.entity.EntityEndpointImpl;
+import de.hpi.bpt.argos.api.entityMapping.EntityMappingEndpointImpl;
+import de.hpi.bpt.argos.api.entityType.EntityTypeEndpointImpl;
+import de.hpi.bpt.argos.api.eventQuery.EventQueryEndpointImpl;
+import de.hpi.bpt.argos.api.eventType.EventTypeEndpointImpl;
+import de.hpi.bpt.argos.common.EventProcessingPlatformUpdaterImpl;
+import de.hpi.bpt.argos.common.RestEndpoint;
+import de.hpi.bpt.argos.eventProcessing.EventReceiver;
+import de.hpi.bpt.argos.eventProcessing.EventReceiverImpl;
+import de.hpi.bpt.argos.eventProcessing.mapping.EventEntityMapper;
+import de.hpi.bpt.argos.eventProcessing.mapping.EventEntityMapperImpl;
+import de.hpi.bpt.argos.eventProcessing.status.EntityStatusCalculatorImpl;
+import de.hpi.bpt.argos.notifications.ClientUpdateServiceImpl;
+import de.hpi.bpt.argos.parsing.EventTypeParserImpl;
+import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
+import de.hpi.bpt.argos.util.LoggerUtilImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Service;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public class ArgosImpl implements Argos {
+	private static final Logger logger = LoggerFactory.getLogger(ArgosImpl.class);
+	private Service sparkService;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void start() {
+		sparkService = Service.ignite()
+				.port(Argos.getPort())
+				.threadPool(Argos.getThreads())
+				.staticFileLocation(Argos.getPublicFiles());
+
+		if (!PersistenceAdapterImpl.getInstance().establishConnection()) {
+			stop();
+			return;
+		}
+
+		// TODO: parse static data

```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1135)